pom.xml

```diff
@@ -32,8 +32,8 @@
 
     <licenses>
         <license>
-            <name>GNU General Public License</name>
-            <url>http://www.gnu.org/licenses/gpl.html</url>
+            <name>GNU Lesser General Public License</name>
+            <url>http://www.gnu.org/licenses/lgpl-2.1.html</url>
```

Comment:
You're right this is a lgpl instead of gpl but this is the version 3 and not the version 2.1. Could you change it ?